### PR TITLE
Double Click

### PR DESCRIPTION
**Pull request by bfox6**
- Double clicking on a marker will zoom in on that marker's bounds center and zoom in 2 levels

### CODE REVIEW

File: src/map/BoundsDriftMarker.tsx [hunk 1]
```diff
@@ -1,32 +1,45 @@
...
   const [displayBounds, setDisplayBounds] = useState<boolean>(false)
+  const { map } = useLeaflet();
+  const boundingBox = new LatLngBounds([
+      [bounds.southWest.lat, bounds.southWest.lng], [bounds.northEast.lat, bounds.northEast.lng]])
 
   // DriftMarker misbehaves if icon=undefined is provided
   // is this the most elegant way?
   const optionalIconProp = icon ? { icon } : { };
 
+  const HandleDoubleClick = () => {
```
Comment (bobular) [inline src/map/BoundsDriftMarker.tsx:24]: I think we use lowercase leading 'h' for methods?

File: src/map/MarkerBounds.stories.tsx [hunk 2]
```diff
@@ -64,3 +65,27 @@ export const MarkerBounds = () => {
       />
   );
 };
+
+export const DoubleClick = () => {
```
Comment (bobular) [inline src/map/MarkerBounds.stories.tsx:69]: Is this story any different from the first?

**Review by bobular — COMMENTED** [review]
Works great in the Marker Bounds story :-)

I assumed this would "just work" in the donut and chart stories, but it seems to freeze (and make a large grey background) - could you investigate please?

**Review by bobular — COMMENTED** [review]
Actually - did you try the fitBounds() approach?  I see we're zooming to the center +2 zoom.

I thought it was already doing that (the first few markers I clicked on seemed to zoom appropriately) - but it isn't.  Could you comment please?

File: src/map/MarkerBounds.stories.tsx (hunk 2 above)
Comment (bfox6) [reply src/map/MarkerBounds.stories.tsx:69]: I meant to delete this 😆

**Comment by bfox6** [comment]
I switched to fitBounds and it works much better. It also for whatever reason solved the errors with the donut and chart stories 🥳

**Comment by bobular** [comment]
That's great.  I pushed some small changes. 
Seems to work for all the stories that use BoundsDriftMarker - nice!
Can merge any time, but let's let Dave cast his eye over it perhaps?

**Review by dmfalke — APPROVED** [review]
I looked at the changes, and all looks good to me!

**Pull request merged**